python/tests/ingestion/test_dataset_utils.py

```diff
@@ -25,6 +26,21 @@
     ['04', 'Arizona', 'UNKNOWN', '10'],
 ]
 
+_fake_race_data_some_only_all = [
```

Comment:
Is this used?

Reply:
nope; I had deleted them but i think they were resurrected in the merge process.. thanks